include/Inventor/SbByteBuffer.h

```diff
@@ -73,7 +73,7 @@ class COIN_DLL_API SbByteBuffer {
 
 #endif // !COIN_SBBYTEBUFFER_H
 
-//The SBBYTEBUFFER_PRIVATE_VARIABLES must survice an inclusion from the .icc file
+//The SBBYTEBUFFER_PRIVATE_VARIABLES must service an inclusion from the .icc file
```

Comment:
I guess this should rather read `survive` than `service`.

Reply:
Corrected in [759bacd](https://github.com/coin3d/coin/pull/486/commits/759bacdeef2e4ab87dd1d8a9d86c9ce7c6fcb662)